types/react-pdf/dist/Page.d.ts

```diff
@@ -0,0 +1,191 @@
+import * as React from 'react';
+
+export type RenderFunction = () => JSX.Element;
+
+/* tslint:disable:ban-types */
+export interface PDFPageItem {
+    _transport: object;
+    commonObjs: object;
+    getAnnotations: Function;
```

Comment:
you might consider `(...args: any[]) => any` instead of `Function` if you haven't already. It's almost the same, but doesn't accept `Function`, which might be desired.